vhost-user-backend/src/event_loop.rs

```diff
@@ -57,6 +58,44 @@ impl std::error::Error for VringPollError {}
 /// Result of vring epoll operations.
 pub type VringEpollResult<T> = std::result::Result<T, VringPollError>;
 
+bitflags! {
+    #[derive(Debug, PartialEq, PartialOrd)]
+    pub struct VringPollEvent: u32 {
+        const READABLE = 1u32;
+        const WRITABLE = 2u32;
+    }
+}
+
+impl From<Interest> for VringPollEvent {
+    fn from(value: Interest) -> Self {
+        let mut event = VringPollEvent::empty();
+        if value.is_readable() {
+            event |= VringPollEvent::READABLE;
+        }
+        if value.is_writable() {
+            event |= VringPollEvent::WRITABLE;
+        }
+        return event;
+    }
+}
```

Comment:
doesn't this expose the mio::Interest type?, since VringPollEvent this impl is also public. Maybe just a private function is enough

Reply:
This one is unecessary. I will delete it.